plugins/inventory/vmware_vm_inventory.py

```diff
@@ -552,6 +558,24 @@ def to_flatten_dict(d, parent_key='', sep='.'):
     return dict(items)
 
 
+def rename_dict_key(item, rename_func):
+    """
+    Parse properties dict with rename_func
+
+    """
+    if isinstance(item, dict):
+        new_dict = {}
+        for k, v in item.items():
+            new_dict[rename_func(k)] = rename_dict_key(v, rename_func)
+        return new_dict
+    elif isinstance(item, list):
+        new_list = []
+        for k in item:
+            new_list.append(rename_dict_key(k, rename_func))
```

Comment:
I think `return` is missing here -

```suggestion
            new_list.append(rename_dict_key(k, rename_func))
        return new_list
```
Correct me if I am wrong.

Reply:
👍